localplanning.services/src/layouts/Layout.astro

```diff
@@ -0,0 +1,34 @@
+---
+import { ClientRouter } from "astro:transitions";
+import "../styles/global.css";
+import NavBar from "../components/NavBar.astro";
+import Footer from "../components/Footer.astro";
+---
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <ClientRouter />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
+    <meta name="generator" content={Astro.generator} />
+    <title>localplanning.services</title>
+  </head>
+  <body class="bg-main">
+    <NavBar />
+    <main>
+      <slot />
+    </main>
+    <Footer />
+  </body>
+</html>
+
+<style>
+  html,
+  body {
+    margin: 0;
+    width: 100%;
+    height: 100%;
+  }
+</style>
```

Comment:
Styles can be applied within a component scoped `<style/> tag, or inline with classes.

Docs - https://docs.astro.build/en/guides/styling/